.github/ISSUE_TEMPLATE/01_bug.yaml

```diff
@@ -0,0 +1,34 @@
+name: Bug report
+description: If something isn't working
+title: "[BUG] <title>"
+labels: needs-review
```

Comment:
Maybe instead of prefixing this with `[BUG]` we can simply add a label `bug`?

Reply:
@matthiasbeyer : updated label as `bug` and removed `[BUG]`  from prefix.

Reply:
you can try out behaviour at https://github.com/hgvanpariya/enterprise-rust-template/issues